src/promptflow/tests/test_configs/flows/llm_connection_override/flow.dag.yaml

```diff
@@ -0,0 +1,21 @@
+inputs:
+  text:
+    type: string
+outputs:
+  output_prompt:
+    type: string
+    reference: ${summarize_text_content_prompt.output}
+nodes:
+- name: summarize_text_content_prompt
+  type: llm
```

Comment:
maybe use a dummy python tool here instead of the llm one, so we don't need to ignore the serving test in live mode.

Reply:
this sample is for remote connection, seems there's no difference between a python tool and llm tool in this scenario?